autopush/main.py

```diff
@@ -80,20 +78,16 @@ def add_shared_args(parser):
                         type=int, default=5, env_var="ROUTER_WRITE_THROUGHPUT")
     parser.add_argument('--log_level', type=int, default=40,
                         env_var="LOG_LEVEL")
-    parser.add_argument(
-        '--max_data',
-        help="Max data segment length in bytes",
-        default=4096,
-        env_var='MAX_DATA')
+    parser.add_argument('--max_data', help="Max data segment length in bytes",
+                        default=4096, env_var='MAX_DATA')
 
 
-def add_bridge_args(parser):
+def add_router_args(parser):
```

Comment:
We're mixing terms here. We should remove "bridge" if this is simply another form of routing, otherwise we'll have too many synonyms.